src/components/form/PillInput.res

```diff
@@ -117,34 +117,27 @@ let make = (~name, ~initialItems: array<string>=[], ~placeholder, ~duplicateChec
     <div className="w-full flex flex-wrap gap-2 border p-2 text-sm rounded-md">
       {items
       ->Array.mapWithIndex((item, i) =>
-        switch editingItem {
-        | Some(selected) =>
-          <RenderIf condition={selected == item}>
-            <div key={Int.toString(i)} className="flex flex-wrap gap-1 p-1 border rounded-md">
-              <input
-                type_="text"
-                value={editInput}
-                onBlur={_ => saveItem(item)}
-                onInput=handleEditChange
-                onKeyDown={ev => handleEditKeydown(item, ev)}
-                className="rounded-md p-1 flex-grow"
-              />
-              <button onClick={_ => removeItem(item)} className=" hover:bg-gray-300 rounded-md p-1">
-                <Icon name="cross-outline" size=14 />
-              </button>
-            </div>
+        <div key={Int.toString(i)} className="flex flex-wrap gap-1 p-1 border rounded-md">
+          <RenderIf condition={editingItem == Some(item)}>
```

Comment:
```suggestion
          <RenderIf condition={editingItem->Option.isSome}>
```

Reply:
here I am not checking if it has some value or not
I am doing direct comparision to check if the current `editingItem` is equal to the specific item